test/unit/Site.test.js

```diff
@@ -351,6 +353,69 @@ test('Site read correct user defined variables', async () => {
   expect(subsub.number).toEqual('9999');
 });
 
+test('Site convert generates correct assets', async () => {
+  const json = {
+    'src/asset/font-awesome.csv': '',
+    'src/asset/glyphicons.csv': '',
+    'src/template/page.ejs': PAGE_EJS,
+    'inner/site.json': SITE_JSON_DEFAULT,
+
+    'asset/css/bootstrap.min.css': '',
+    'asset/css/bootstrap.min.css.map': '',
+    'asset/css/github.min.css': '',
+    'asset/css/markbind.css': '',
+    'asset/css/page-nav.css': '',
+    'asset/css/site-nav.css': '',
+
+    'asset/js/bootstrap-utility.min.js': '',
+    'asset/js/setup.js': '',
+    'asset/js/vue.min.js': '',
+    'asset/js/vue-strap.min.js': '',
+
+    'node_modules/@fortawesome/fontawesome-free/css/all.min.css': '',
+    'node_modules/@fortawesome/fontawesome-free/webfonts/font1.svg': '',
+    'node_modules/@fortawesome/fontawesome-free/webfonts/font2.ttf': '',
+
+    'inner/_markbind/layouts/default/footer.md': '',
+    'inner/_markbind/layouts/default/head.md': '',
+    'inner/_markbind/layouts/default/header.md': '',
+    'inner/_markbind/layouts/default/navigation.md': '',
+    'inner/_markbind/layouts/default/scripts.js': '',
+    'inner/_markbind/layouts/default/styles.css': '',
```

Comment:
I think you should test that the original site was just a wiki and then you call `init` and subsequently `convert`. No need for `generate`.

Reply:
Has this already been addressed? Resolve this conversation if it has.